compiler/examples/gimli/proofs/Makefile

```diff
@@ -2,33 +2,27 @@ ECARGS ?= -I Jasmin:../../../../eclib
 
 JASMIN2EC := ../../../jasmin2ec
 
-CT_MODE ?= CT
-GIMLI_CT = gimli_$(shell echo $(CT_MODE) | tr A-Z a-z).ec
-
 .SUFFIXES: .jazz .ec
 
-all: ct gimli_arm.ec gimli_x86.ec gimli_x86_ct.ec gimli_arm_ct.ec gimli_avx.ec gimli_avx_ct.ec
+all: gimli_ct.ec gimli_arm.ec gimli_x86.ec gimli_x86_ct.ec gimli_arm_ct.ec gimli_avx.ec gimli_avx_ct.ec
```

Comment:
gimli_ct.ec is not a target. Could you also use the same order for the dependencies of `all` and for the declaration of the targets. It helps checking that everything is fine.

Reply:
I think I’ve addressed your comment: removed `gimli_ct.ec` from the list & sort it according to the order below.